pkg/agent/route/route_linux.go

```diff
@@ -657,6 +657,14 @@ func (c *Client) DeleteRoutes(podCIDR *net.IPNet) error {
 	return nil
 }
 
+func (c *Client) AddServiceRoutes(svcIP net.IP, gwIP net.IP) error {
+	return nil
+}
+
+func (c *Client) DeleteServiceRoutes(svcIP net.IP) error {
+	return nil
+}
```

Comment:
maybe return an error here (e.g. `fmt.Errorf("AddServiceRoutes is not implemented on Linux")`) as this function should not be called at all

Reply:
Updated.